_maps/map_files/Pahrump/Pahrump-Surface-2.dmm

```diff
@@ -20917,7 +20917,6 @@
 /obj/item/clothing/gloves/combat,
 /obj/item/clothing/shoes/combat,
 /obj/item/storage/belt/military/army,
- /obj/item/melee/onehanded/knife/hunting,
```

Comment:
Are these knife removals intentional changes?

Reply:
Automatic. Might've been a recent change that I accidentally reverted.